sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -1494,9 +1494,7 @@ def test_neighbors_badargs():
         est.fit(X)
 
     for cls in (
-        neighbors.KNeighborsClassifier,
         neighbors.RadiusNeighborsClassifier,
-        neighbors.KNeighborsRegressor,
```

Comment:
@glemaitre and @jeremiedbb: shouldn't we keep those for this test?

Reply:
removing them means that we are testing the estimators. So we need to remove from the list.

Reply:
Don't we want to check for those checks which are not done in the parameter validation step?

Reply:
Oh I see, I misread that it was not the `test_commons.py` file.
Ideally, we should only remove tests that are covered by the common test. Since they are missed. It would be nice to split the test in to 2 tests from which one can be removed in the future.

Reply:
I've opened https://github.com/scikit-learn/scikit-learn/pull/23968 to revert this.

Reply:
Thanks. I will have a look at it.